.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,9 +7,9 @@ assignees: ''
 
 ---
 
-# Describe the problem that this feature improves
+# What's the problem that this feature alleviates?
```

Comment:
not sure this helps non-native speakers ⛺️

Reply:
`What's the problem that this feature ameliorates?` 😄
`What's the problem that this feature fixes?` 
`What problem does this feature fix?` 
`What's the problem?`

Reply:
What problem would this feature fix?